addons/binding/org.openhab.binding.neeo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="neeo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- The brain bridge - will have things dynamically assigned to it. -->
+	<bridge-type id="brain">
```

Comment:
BTW - this was an EXCELLENT exercise you had me do.  I discovered an issue that has been bugging me for awhile in how I was dynamically generating the macros - was a minor issue so I never put the time into it - but doing this I found the issue (and realized it may be more than a 'minor' issue in certain circumstances).